frontend/components/issues/js/issues.component.ts

```diff
@@ -233,22 +232,21 @@ class IssuesController implements ng.IController {
 
 				this.issuesReady.then(() => {
 					const hasPerm = this.AuthService.hasPermission(
-						this.ClientConfigService.permissions.PERM_CREATE_ISSUE, 
-						this.modelSettings.permissions
+						this.ClientConfigService.permissions.PERM_CREATE_ISSUE,
```

Comment:
````js
this.canAddIssue = this.AuthService.hasPermission(....)
````

Does this have latest of staging? I feel like we've changed this already on a separate branch

Reply:
That was for groups :P The reason this code is even highlighted is auto formatting by TSLint

Reply:
no I mean i thought we've got rid of the hasPerm variable and just assigned this.canAddIssue directly somewhere already. I remember pointing this out elsewhere

Reply:
Yes it was very similar looking code but different file;  ```this.canAddGroup```  vs ```this.canAddIssue```. Probably worth changing it here to, I can look into it.